src/main/kotlin/no/nav/familie/ks/sak/kjerne/behandling/steg/vedtak/Begrunnelse.kt

```diff
@@ -0,0 +1,75 @@
+package no.nav.familie.ks.sak.kjerne.behandling.steg.vedtak
+
+import no.nav.familie.eksterne.kontrakter.AnnenForeldersAktivitet
+import no.nav.familie.eksterne.kontrakter.SøkersAktivitet
+import no.nav.familie.ks.sak.kjerne.vedtak.VedtakBegrunnelseType
+
+enum class Begrunnelsetype {
+    STANDARD_BEGRUNNELSE,
+    EØS_BEGRUNNELSE,
+    FRITEKST
+}
+
+interface Begrunnelse : Comparable<Begrunnelse> {
+    val type: Begrunnelsetype
+    val vedtakBegrunnelseType: VedtakBegrunnelseType?
+
+    override fun compareTo(other: Begrunnelse): Int {
+        return when {
+            this.type == Begrunnelsetype.FRITEKST -> Int.MAX_VALUE
+            other.type == Begrunnelsetype.FRITEKST -> -Int.MAX_VALUE
+            this.vedtakBegrunnelseType == null -> Int.MAX_VALUE
+            other.vedtakBegrunnelseType == null -> -Int.MAX_VALUE
+
+            else -> this.vedtakBegrunnelseType!!.sorteringsrekkefølge - other.vedtakBegrunnelseType!!.sorteringsrekkefølge
+        }
+    }
+}
+
+interface BegrunnelseMedData : Begrunnelse {
+    val apiNavn: String
+}
+
+data class BegrunnelseData(
+    override val vedtakBegrunnelseType: VedtakBegrunnelseType,
+    override val apiNavn: String,
+
+    val gjelderSoker: Boolean,
+    val barnasFodselsdatoer: String,
+    val fodselsdatoerBarnOppfyllerTriggereOgHarUtbetaling: String,
+    val fodselsdatoerBarnOppfyllerTriggereOgHarNullutbetaling: String,
+    val antallBarn: Int,
+    val antallBarnOppfyllerTriggereOgHarUtbetaling: Int,
+    val antallBarnOppfyllerTriggereOgHarNullutbetaling: Int,
+    val maanedOgAarBegrunnelsenGjelderFor: String?,
+    val maalform: String,
+    val belop: String,
+    val soknadstidspunkt: String,
+    val avtaletidspunktDeltBosted: String,
+    val sokersRettTilUtvidet: String
```

Comment:
```suggestion
    val gjelderSøker: Boolean,
    val barnasFødselsdatoer: String,
    val fødselsdatoerBarnOppfyllerTriggereOgHarUtbetaling: String,
    val fødselsdatoerBarnOppfyllerTriggereOgHarNullutbetaling: String,
    val antallBarn: Int,
    val antallBarnOppfyllerTriggereOgHarUtbetaling: Int,
    val antallBarnOppfyllerTriggereOgHarNullutbetaling: Int,
    val månedOgÅrBegrunnelsenGjelderFor: String?,
    val målform: String,
    val beløp: String,
    val søknadstidspunkt: String,
    val avtaletidspunktDeltBosted: String,
    val søkersRettTilUtvidet: String
```

Lurer på om vi bør kjøre på for fullt med Æ Ø Å nå. 
Litt forskjellig hva vi bruker i koden nå, men jo raskere vi går for oo eller ø, jo bedre 🙌 

Gjelder også de andre klassene

Reply:
Sant!

Reply:
Har sett litt på dette og tror ikke vi kan endre dette så lett. Ihvertfall ikke om vi ikke også endrer på dto-er i `familie-brev`. `familie-brev` brukes av andre enn oss så er litt skeptisk til å gjøre alt for store endringer her 🤔